readme/template-guide-readme.md

```diff
@@ -0,0 +1,101 @@
+# README Template Guide
+
+Read this document before you start working on the README template.
+
+
+*Note: As your project evolves and grows, so should your README file. READMEs should be revisited regularly to ensure relevant information is captured for new users.*
+
+## Why do I need a README?
```

Comment:
Same as above. I would avoid the personal pronoun:
Why is a README important?

Reply:
The Good Docs project's outline for template guides include this heading, so we'll keep it for consistency across the project's templates.